gettingstarted/settings.py

```diff
@@ -105,15 +127,41 @@
 
 USE_I18N = True
 
-USE_L10N = True
-
 USE_TZ = True
 
 
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/2.0/howto/static-files/
 
-STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
+STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
 STATIC_URL = "/static/"
```

Comment:
(To match what `django-admin startproject` generates) 

```suggestion
STATIC_URL = "static/"
```